server/src/main.rs

```diff
@@ -0,0 +1,96 @@
+//! svc-cargo
+//! Processes flight requests from client applications
+
+use axum::{handler::Handler, routing, Router, Server};
+use hyper::Error;
+use std::net::{Ipv4Addr, SocketAddr};
+use utoipa::OpenApi;
+// use utoipa_swagger_ui::SwaggerUi;
+
+mod rest;
+
+/// Tokio signal handler that will wait for a user to press CTRL+C.
+/// We use this in our hyper `Server` method `with_graceful_shutdown`.
+///
+/// # Arguments
+///
+/// # Examples
+///
+/// ```
+/// Server::bind(&"0.0.0.0:8000".parse().unwrap())
+/// .serve(app.into_make_service())
+/// .with_graceful_shutdown(shutdown_signal())
+/// .await
+/// .unwrap();
+/// ```
+async fn shutdown_signal() {
+    tokio::signal::ctrl_c()
+        .await
+        .expect("expect tokio signal ctrl-c");
+    println!("signal shutdown!");
+}
+
+/// Responds a NOT_FOUND status and error string
+///
+/// # Arguments
+///
+/// # Examples
+///
+/// ```
+/// let app = Router::new()
+///         .fallback(not_found.into_service());
+/// ```
+pub async fn not_found(uri: axum::http::Uri) -> impl axum::response::IntoResponse {
+    (
+        axum::http::StatusCode::NOT_FOUND,
+        format!("No route {}", uri),
+    )
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Error> {
+    let rest_port = std::env::var("HOST_PORT")
```

Comment:
This should either map to the DOCKER_PORT or just stay hardcoded to 8000.
It's the port the server will listen on inside the container. If we run docker, we map the DOCKER_PORT to a unique HOST_PORT so we can use that for our routing with istio (or any other application loadbalancer)

Reply:
Fixed https://github.com/Arrow-air/svc-cargo/pull/1/commits/6ba3b8a3f9550236538b753d150cbb303d2bc388